lib/segment/tests/integration/hnsw_quantized_search_test.rs

```diff
@@ -156,7 +157,9 @@ fn hnsw_quantized_search_test(
         );
         let best_2 = oversampling_2_result[0][0];
 
-        assert!(best_2.idx == best_1.idx || best_2 >= best_1);
+        let acceptable_range = (best_1.score - oversampling_1_result[0][1].score).abs();
```

Comment:
Could you please explain this part?

Reply:
I didn't want to have a magic number as an acceptable epsilon. As epsilon, I choose the difference between 2 best points. While debugging I noticed that this difference is pretty small

Reply:
Thanks for the explanation.

Btw, there's also `abs_diff`:

```suggestion
        let acceptable_range = best_1.score.abs_diff(oversampling_1_result[0][1].score);
```

Reply:
Why would we need to use the epsilon in a first place?

Reply:
@generall proposed a better check without approximate comparison. resolve this conversation